master/getting-started/rkt/tutorials/basic.md

```diff
@@ -0,0 +1,206 @@
+---
+title: Calico networking with rkt
+---
+
+
+This tutorial describes how to set up a Calico cluster in a pure rkt environment
+using CNI with Calico specific network and IPAM drivers.
+
+Using the Calico CNI plugin, the required network setup and configuration
+for networking containers using Calico is handled automatically as part of the
+standard network and container lifecycle.  Provided the network is created
+using the Calico driver, creating a container using that network will
+automatically add the container to the Calico network, creating all necessary
+Calico configuration and setting up the interface and routes in the container
+accordingly.
+
+The Calico IPAM driver must be used in addition to the the Calico network
+driver.  This provides IP address management using the configured Calico IP
+Pools as address pools for the container, preferentially selecting sub-blocks
+of IPs for a particular host.
+
+## 1. Environment setup
+
+To run through the worked example in this tutorial you will need to set up two hosts
+with a number of installation dependencies.
+
+Follow the instructions in the tutorial below to set up a virtualized
+environment using Vagrant - be sure to follow the appropriate instructions
+for _Running the Calico rkt tutorials on CoreOS using Vagrant and VirtualBox_.
+
+- [Vagrant install with CoreOS]({{site.baseurl}}/{{page.version}}/getting-started/rkt/installation/vagrant-coreos/)
+
+
+If you have everything set up properly you should have `calicoctl` in your
+`$PATH`, and two hosts called `calico-01` and `calico-02`.
```

Comment:
Can we do something about the fact that we rely on the hosts being called `calico-01` and `calico-02`?
 
If possible I'd really like this guide to be something people can follow more generally.